Formula/scipy.rb

```diff
@@ -47,7 +82,17 @@ def install
     Pathname("site.cfg").write config
 
     version = Language::Python.major_minor_version Formula["python@3.9"].opt_bin/"python3"
-    ENV["PYTHONPATH"] = Formula["numpy"].opt_lib/"python#{version}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", Formula["cython"].opt_libexec/"lib/python#{version}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", Formula["numpy"].opt_lib/"python#{version}/site-packages"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{version}/site-packages"
+
+    resources.each do |r|
+      r.stage do
```

Comment:
Do we not want to install these into a virtualenv in `libexec`? How does `numpy` know how to find the vendored resources?

Reply:
`pythran` should be similar to `cython` in that both are compilers.
https://scipy.github.io/devdocs/building/macosx.html#compilers-c-c-fortran-cython-pythran

I guess we could consider a separate formula, though I didn't see any other formulae using it as dependency.

Reply:
Sorry, that was a typo. I meant `scipy`. I was wondering how `scipy` would find these resources at runtime.

Reply:
I believe it is just needed during build time.

Reply:
If these are needed only for the build then we should install them into `buildpath`. I'm not opposed to a separate `pythran` formula too, since I think it can substantially simplify this one.

Reply:
I'll probably look into a separate Formula.

From quick lookup, I see a couple Linux distros that have `pythran`, e.g.
- ArchLinux: https://archlinux.org/packages/community/any/python-pythran/
- Fedora: https://src.fedoraproject.org/rpms/pythran